src/components/TipInput.vue

```diff
@@ -130,25 +130,33 @@ export default {
       return (this.value > 0 && (this.isRetip || this.isMessageValid)) || this.isMessageValid;
     },
     isTipped() {
-      return !this.loading
-        || (this.tip.sender === this.account)
-        || this.tip.retips.filter((retip) => retip.sender === this.account).length > 0;
+      if (!this.stats || !this.derivedTipStats) {
+        return false;
+      }
+      return this.derivedTipStats.senders.find((sender) => sender === this.account);
```

Comment:
this should return boolean 
```suggestion
      return this.derivedTipStats.senders.find((sender) => sender === this.account) !== null;
```
or hackish
```suggestion
      return !!this.derivedTipStats.senders.find((sender) => sender === this.account);
```